pages/VerifiableCredentials/Types/VerifiableCredentialSchema.md

```diff
@@ -0,0 +1,182 @@
+# Verifiable Credential Schema
+
+The schema for a DSNP Verifiable Credential MAY be defined using the format described in the [Verifiable Credentials JSON Schema](https://www.w3.org/TR/vc-json-schema/) W3C Candidate Recommendation Draft.
+This format provides a credential wrapper around a JSON schema document.
+
+Empty schemas (Verifiable Credential Schema Documents with `"jsonSchema": {}`) are allowed; however, schemaless credentials may be preferred in this situation.
+Empty schemas are useful in situations where no attribute data fields are relevant but the schema author wishes to assert authorship.
+
+## Verifiable Credential Schema Document
+
+A Verifiable Credential Schema document is a JSON-LD document that is itself a [Verifiable Credential](./VerifiableCredential.md) with a claim that includes the schema and (optionally) the DSNP extension described below.
+A schema credential specializes the meaning of the following fields:
+
+| Property | Required | JSON Type | Description | Restrictions |
+| --- | --- | --- | --- | --- |
+| `type` | YES | Array of strings | Type of credential | MUST contain the strings `"VerifiableCredential"` and `"JsonSchemaCredential"` |
+|`credentialSubject` | YES | Object | Object containing JSON schema and DSNP extensions | See [Credential Subject](#credential-subject) |
```

Comment:
This link does not work.

Reply:
Working for me

Reply:
I think these are only an issue when looking at the github renderings; the published docs are OK.